src/main/java/net/fortuna/ical4j/model/Rfc5545Rule.java

```diff
@@ -0,0 +1,27 @@
+package net.fortuna.ical4j.model;
+
+/**
+ * Incarnation of a RFC5545 rule.
+ * 
+ * @author daniel grigore
+ * @author corneliu dobrota
+ *
+ * @param <T>
+ *            type of the element this rule can be applied to
+ */
+public interface Rfc5545Rule<T> {
```

Comment:
@danielgrigore I think these transformation rules would be better suited under the net.fortuna.ical4j.transform package. This package has long been neglected but was intended for applying mutations to calendar/component/property objects.